docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Exciting MUI X improvements: v6.11.0
+description: Support for time zones, Charts in alpha, Data Grid filtering, and more.
```

Comment:
https://en.wiktionary.org/wiki/timezone#:~:text=zone%20(chiefly%20US)-,Usage%20notes,likely%20to%20use%20time%20zone.

> In the US, [timezone](https://en.wiktionary.org/wiki/timezone#English) seems to be favored considerably (unless comparing different specific zones), while the UK seems more likely to use [time zone](https://en.wiktionary.org/wiki/time_zone#English).

In the doc I use `timezones`
Do you have a strong preference for `time zones` ?
If so, we should probably unify everywhere

Reply:
I prefer `timezones`.

cc: @samuelsycamore to get his thoughts on this as well.